### ISSUE TITLE
Getting ReferenceError: self is not defined with 3.3.3

### ISSUE THREAD

**Issue by tmtk75** [moroshko/autosuggest-highlight#26]
Hello, I use autosuggest-highlight with Next.js.

I'm getting this error after I bumped up the module to 3.3.3.
```
error - ReferenceError: self is not defined
    at Object.<anonymous> (/the/path/to/node_modules/autosuggest-highlight/match/index.js:1:232)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.autosuggest-highlight/match (/a/path/to/.next/server/pages/[id].js:1935:18)
    at __webpack_require__ (/a/path/to/.next/server/webpack-runtime.js:33:42)
    at eval (webpack-internal:///./components/modules/a-component/index.tsx:14:85) {
  page: '/[id]'
}
```

I checked `./node_modules/autosuggest-highlight/match/index.js` of 3.3.3 and it has a `self` like this.
```javascript
! function(e, t) {
    "object" == typeof exports && "object" == typeof module ? module.exports = t() : "function" == typeof define && define.amd ? define([], t) : "object" == typeof exports ? exports.AutosuggestHighlightMatch = t() : e.AutosuggestHighlightMatch = t()
}(self, (() => {
    return e = {
...
```

**Comment by dmitry-kovalev** [comment]
Hello

I apologize. Webpack build fixed in 3.3.4.

**Comment by tmtk75** [comment]
It worked with 3.3.4.
Thanks a lot for your quick fix!
